skills/okf/scripts/embed.py

```
@@ -0,0 +1,236 @@
+def fake_embed(texts: list[str], dim: int) -> list[list[float]]:
+    vectors: list[list[float]] = []
+    for text in texts:
+        seed = sum((index + 1) * ord(char) for index, char in enumerate(text))
+        raw = [float(((seed + i * 37) % 101) + 1) for i in range(dim)]
+        vectors.append(l2_normalize(raw))
+    return vectors
+
+
+def post_json(url: str, payload: dict[str, Any], headers: dict[str, str]) -> dict[str, Any]:
+    body = json.dumps(payload).encode("utf-8")
+    req = request.Request(url, data=body, headers={"content-type": "application/json", **headers}, method="POST")
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)